pyproject.toml

```diff
@@ -1,3 +1,34 @@
+[project]
+name = "readme_renderer"
+version = "41.0"
+description = '''readme_renderer is a library for rendering "readme" ' "descriptions for Warehouse'''
+authors = [
+  {email = "admin@mail.pypi.org"},
+  {name = "The Python Packaging Authority"}
+]
+readme = "README.rst"
+license = {file = "LICENSE"}
+dependencies = ["bleach>=2.1.0", "docutils>=0.13.1", "Pygments>=2.5.1"]
+dynamic = ["classifiers"]
+
+[project.optional-dependencies]
+md = ["cmarkgfm>=0.8.0"]
+
+[tool.poetry.dependencies]
+python = ">=3.8"
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏are we also using poetry tool, or is this a copy/paste from another example?
See https://setuptools.pypa.io/en/latest/userguide/dependency_management.html#python-requirement for setuptools syntax